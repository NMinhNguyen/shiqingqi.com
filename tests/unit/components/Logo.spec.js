import { shallowMount } from "@vue/test-utils";
import Logo from "@/components/Logo.vue";

describe("Logo", () => {
  it("renders svg logo", () => {
    expect(shallowMount(Logo)).toMatchSnapshot();
  });

  it("renders logo with different color", () => {
    expect(
      shallowMount(Logo, { propsData: { color: "#878787" } })
    ).toMatchSnapshot();
  });
});
