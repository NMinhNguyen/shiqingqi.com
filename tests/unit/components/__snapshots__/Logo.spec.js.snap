// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders logo with different color 1`] = `
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200" viewBox="0 0 200 200" class="logo" style="isolation: isolate;">
  <defs>
    <clipPath id="_clipPath_t17SoiwPLhbLaPc33JoYkyIOgam8CkEd">
      <rect width="200" height="200"></rect>
    </clipPath>
  </defs>
  <g clip-path="url(#_clipPath_t17SoiwPLhbLaPc33JoYkyIOgam8CkEd)">
    <path fill="#878787" fill-rule="evenodd" d=" M 62.08 0 L 26.667 0 C 11.949 0 0 11.949 0 26.667 L 0 68.585 Q 1.902 72.192 4.333 75.244 L 4.333 75.244 L 4.333 75.244 Q 9.556 81.8 17.111 85.356 L 17.111 85.356 L 17.111 85.356 Q 24.667 88.911 34.444 88.911 L 34.444 88.911 L 34.444 88.911 Q 44 88.911 51.667 85.356 L 51.667 85.356 L 51.667 85.356 Q 59.333 81.8 64.556 75.244 L 64.556 75.244 L 64.556 75.244 Q 69.778 68.689 72.556 59.578 L 72.556 59.578 L 72.556 59.578 Q 75.333 50.467 75.333 39.133 L 75.333 39.133 L 75.333 39.133 Q 75.333 27.8 72.556 18.578 L 72.556 18.578 L 72.556 18.578 Q 69.778 9.356 64.556 2.8 L 64.556 2.8 L 64.556 2.8 Q 63.38 1.324 62.08 0 Z  M 103.228 0 L 137.778 0 L 137.778 116.244 L 170.889 116.244 L 170.889 0 L 173.333 0 C 188.051 0 200 11.949 200 26.667 L 200 173.333 C 200 188.051 188.051 200 173.333 200 L 26.667 200 C 11.949 200 0 188.051 0 173.333 L 0 112.367 Q 10.371 116.938 22.222 118.467 L 22.222 118.467 L 22.222 118.467 Q 24.889 127.8 29.889 134.911 L 29.889 134.911 L 29.889 134.911 Q 34.889 142.022 43.444 146.911 L 43.444 146.911 L 43.444 146.911 Q 52 151.8 64.333 154.578 L 64.333 154.578 L 64.333 154.578 Q 76.667 157.356 94 158.022 L 94 158.022 L 101.111 132.689 L 101.111 132.689 Q 92.222 132.022 84.556 131.133 L 84.556 131.133 L 84.556 131.133 Q 76.889 130.244 71.111 128.467 L 71.111 128.467 L 71.111 128.467 Q 65.333 126.689 61.556 123.911 L 61.556 123.911 L 61.556 123.911 Q 57.778 121.133 56.667 116.689 L 56.667 116.689 L 56.667 116.689 Q 68.667 113.8 78.667 107.356 L 78.667 107.356 L 78.667 107.356 Q 88.667 100.911 95.889 91.133 L 95.889 91.133 L 95.889 91.133 Q 103.111 81.356 107.111 68.356 L 107.111 68.356 L 107.111 68.356 Q 111.111 55.356 111.111 39.133 L 111.111 39.133 L 111.111 39.133 Q 111.111 19.356 105.111 4.356 L 105.111 4.356 L 105.111 4.356 Q 104.219 2.125 103.228 0 Z "></path>
  </g>
</svg>
`;

exports[`Logo renders svg logo 1`] = `
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200" viewBox="0 0 200 200" class="logo" style="isolation: isolate;">
  <defs>
    <clipPath id="_clipPath_t17SoiwPLhbLaPc33JoYkyIOgam8CkEd">
      <rect width="200" height="200"></rect>
    </clipPath>
  </defs>
  <g clip-path="url(#_clipPath_t17SoiwPLhbLaPc33JoYkyIOgam8CkEd)">
    <path fill="#ffffff" fill-rule="evenodd" d=" M 62.08 0 L 26.667 0 C 11.949 0 0 11.949 0 26.667 L 0 68.585 Q 1.902 72.192 4.333 75.244 L 4.333 75.244 L 4.333 75.244 Q 9.556 81.8 17.111 85.356 L 17.111 85.356 L 17.111 85.356 Q 24.667 88.911 34.444 88.911 L 34.444 88.911 L 34.444 88.911 Q 44 88.911 51.667 85.356 L 51.667 85.356 L 51.667 85.356 Q 59.333 81.8 64.556 75.244 L 64.556 75.244 L 64.556 75.244 Q 69.778 68.689 72.556 59.578 L 72.556 59.578 L 72.556 59.578 Q 75.333 50.467 75.333 39.133 L 75.333 39.133 L 75.333 39.133 Q 75.333 27.8 72.556 18.578 L 72.556 18.578 L 72.556 18.578 Q 69.778 9.356 64.556 2.8 L 64.556 2.8 L 64.556 2.8 Q 63.38 1.324 62.08 0 Z  M 103.228 0 L 137.778 0 L 137.778 116.244 L 170.889 116.244 L 170.889 0 L 173.333 0 C 188.051 0 200 11.949 200 26.667 L 200 173.333 C 200 188.051 188.051 200 173.333 200 L 26.667 200 C 11.949 200 0 188.051 0 173.333 L 0 112.367 Q 10.371 116.938 22.222 118.467 L 22.222 118.467 L 22.222 118.467 Q 24.889 127.8 29.889 134.911 L 29.889 134.911 L 29.889 134.911 Q 34.889 142.022 43.444 146.911 L 43.444 146.911 L 43.444 146.911 Q 52 151.8 64.333 154.578 L 64.333 154.578 L 64.333 154.578 Q 76.667 157.356 94 158.022 L 94 158.022 L 101.111 132.689 L 101.111 132.689 Q 92.222 132.022 84.556 131.133 L 84.556 131.133 L 84.556 131.133 Q 76.889 130.244 71.111 128.467 L 71.111 128.467 L 71.111 128.467 Q 65.333 126.689 61.556 123.911 L 61.556 123.911 L 61.556 123.911 Q 57.778 121.133 56.667 116.689 L 56.667 116.689 L 56.667 116.689 Q 68.667 113.8 78.667 107.356 L 78.667 107.356 L 78.667 107.356 Q 88.667 100.911 95.889 91.133 L 95.889 91.133 L 95.889 91.133 Q 103.111 81.356 107.111 68.356 L 107.111 68.356 L 107.111 68.356 Q 111.111 55.356 111.111 39.133 L 111.111 39.133 L 111.111 39.133 Q 111.111 19.356 105.111 4.356 L 105.111 4.356 L 105.111 4.356 Q 104.219 2.125 103.228 0 Z "></path>
  </g>
</svg>
`;
